packages/logger/src/index.ts

```diff
@@ -1,9 +1,12 @@
-import debug from 'debug';
+// import debug from 'debug';
 import { format, Logger, createLogger, transports } from 'winston';
+import { LoggingWinston } from '@google-cloud/logging-winston';
```

Comment:
This is the package made by Google for winston.  See [here](https://www.npmjs.com/package/@google-cloud/logging-winston).